stetho/src/main/java/com/facebook/stetho/inspector/elements/AbstractChainedDescriptor.java

```diff
@@ -144,4 +144,14 @@ public final void setAttributesAsText(Object element, String text) {
   protected void onSetAttributesAsText(E element, String text) {
     mSuper.setAttributesAsText(element, text);
   }
+
+  @Override
+  @SuppressWarnings("unchecked")
+  public final void getStyleProperties(Object element, StyleAccumulator accumulator) {
+    onGetStyles((E) element, accumulator);
+    mSuper.getStyleProperties(element, accumulator);
+  }
+
+  protected void onGetStyles(E element, StyleAccumulator accumulator) {
```

Comment:
The `on___` methods should be named according to their dual (which is the one that calls it). This should be called `onGetStyleProperties`.